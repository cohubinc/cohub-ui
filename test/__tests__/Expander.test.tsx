import React from "react";
import { mount } from "enzyme";

import { Expander, Typography } from "dist";

it("renders without crashing", async () => {
  const wrapper = mount(
    <Expander
      expandElement={<Typography muted>More</Typography>}
      labelPosition="center"
    >
      <div>test</div>
    </Expander>
  );

  expect(wrapper).toMatchSnapshot();
});
