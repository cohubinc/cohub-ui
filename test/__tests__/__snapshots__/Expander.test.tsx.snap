// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Expander
  expandElement={
    <Typography
      muted={true}
    >
      More
    </Typography>
  }
  labelPosition="center"
>
  <div
    className="flex items-center justify-center pointer w-100 mb-1"
    onClick={[Function]}
  >
    <Typography
      muted={true}
    >
      <Component
        muted={true}
      >
        <span
          className=""
          data-qa="text"
          style={
            Object {
              "color": "hsl(356, 3%, 59%)",
              "fontFamily": "Inter",
              "fontSize": "14px",
              "fontWeight": 400,
              "textAlign": "initial",
              "textTransform": "initial",
            }
          }
        >
          More
        </span>
      </Component>
    </Typography>
  </div>
  <AnimateHeight
    animateOpacity={false}
    animationStateClasses={
      Object {
        "animating": "rah-animating",
        "animatingDown": "rah-animating--down",
        "animatingToHeightAuto": "rah-animating--to-height-auto",
        "animatingToHeightSpecific": "rah-animating--to-height-specific",
        "animatingToHeightZero": "rah-animating--to-height-zero",
        "animatingUp": "rah-animating--up",
        "static": "rah-static",
        "staticHeightAuto": "rah-static--height-auto",
        "staticHeightSpecific": "rah-static--height-specific",
        "staticHeightZero": "rah-static--height-zero",
      }
    }
    applyInlineTransitions={true}
    delay={0}
    duration={250}
    easing="ease"
    height={0}
    style={Object {}}
  >
    <div
      aria-hidden={true}
      className="rah-static rah-static--height-zero"
      style={
        Object {
          "height": 0,
          "overflow": "hidden",
        }
      }
    >
      <div
        style={Object {}}
      >
        <div
          style={
            Object {
              "width": "100%",
            }
          }
        >
          <div>
            test
          </div>
        </div>
      </div>
    </div>
  </AnimateHeight>
</Expander>
`;
